app/views/admin/site_customization/pages/edit.html.erb

```diff
@@ -1,11 +1,11 @@
-<% provide :title do %>
-  <%= t("admin.header.title") %> - <%= t("admin.menu.site_customization.pages") %> - <%= @page.title %>
+<% provide :slug do %>
```

Comment:
Actually, we need to use `provide :title` :wink:, since the parameter here is used to generate the HTML `<title>` tag (see `app/views/layouts/_common_head.html.erb`, the line saying `<title><%= content_for?(:title) ? yield(:title) : default_title  %></title>`).

Reply:
Ok I will check this.  I took the daring to replace `slug`with `title`but I am not sure is the best way to go. 😉 I take this way because slug like status and created,updated at are not translatable in many languages but I could change everything to use `title`again. Just confirm me!